src/Sylius/Bundle/ApiBundle/Resources/config/api_resources/Taxon.xml

```diff
@@ -70,6 +70,13 @@
             </itemOperation>
         </itemOperations>
 
+        <subresourceOperations>
+            <subresourceOperation name="images_get_subresource">
+                <attribute name="method">GET</attribute>
+                <attribute name="path">/admin/taxons/{code}/images</attribute>
```

Comment:
1. True, I've adjusted the summary of this endpoint 👌🏻 
2. That is also true, but I've added a subresource operation instead of adding a filter to the taxon images GET endpoint. It makes a bit of sense to use IRI here, but I'm not entirely convinced it's a good idea to hide this data, plus, it wouldn't be consistent with the product and product images we're now displaying embedded data. But I'm keen to read other opinions. Of course, we could change it for taxon and product images later in a separate PR.

Reply:
2 IMO, I prefer the array with details because I have the path, so I only need one request to get the picture. If we put the IRI into the array, we would need two requests to get the picture(if there is 5 `taxonImages` we need 10 requests).